microcosm_postgres/store.py

```diff
@@ -49,6 +51,10 @@ def assign_model_class_store(self):
             # short cuts while still having an abstraction layer we can replace.
             self.model_class.store = self
 
+    @property
+    def model_name(self):
```

Comment:
None or empty string is not allowed in stats. Return something like “unknown” here

Reply:
Right. In PostgresStoreMetrics, I filter out any calls which does not have model name set: https://github.com/globality-corp/microcosm-postgres/pull/81/files#diff-f7516a258410b943c8f780ce3ee04712R44